examples/hono-edge/wrangler.toml

```diff
@@ -0,0 +1,7 @@
+name = "decart-hono-edge"
+main = "src/index.ts"
+compatibility_date = "2024-11-18"
```

Comment:
best practice is to use today's date so I think we can start with it :) this is a fairly old date